tests/Acceptance/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\BoxNowBundle\Tests\Acceptance\DependencyInjection;
+
+use Answear\BoxNowBundle\ConfigProvider;
+use Answear\BoxNowBundle\DependencyInjection\AnswearBoxNowExtension;
+use Answear\BoxNowBundle\DependencyInjection\Configuration;
+use Matthias\SymfonyConfigTest\PhpUnit\ConfigurationTestCaseTrait;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+class ConfigurationTest extends TestCase
+{
+    use ConfigurationTestCaseTrait;
+
+    /**
+     * @dataProvider provideValidConfig
+     */
+    public function testValidTest(array $configs): void
+    {
+        $this->assertConfigurationIsValid($configs);
+
+        $extension = $this->getExtension();
+
+        $builder = new ContainerBuilder();
+        $extension->load($configs, $builder);
+
+        $configProviderDefinition = $builder->getDefinition(ConfigProvider::class);
+
+        self::assertSame($configs[0]['clientId'], $configProviderDefinition->getArgument(0));
+        self::assertSame($configs[0]['clientSecret'], $configProviderDefinition->getArgument(1));
+
+        if (isset($configs[0]['apiUrl'])) {
+            self::assertSame($configs[0]['apiUrl'], $configProviderDefinition->getArgument(2));
+        }
+    }
+
+    /**
+     * @dataProvider provideInvalidConfig
```

Comment:
we used to using `@test` mark for tests

Reply:
I care about consistency. And we need to do things the way we do on a the team, or there is a mechanim that will make sure thet we will never get `@test` on this project wrote by new contributor.
Need to thing about it.

Reply:
Ok, changed :)